### DISCUSSION TITLE
Restoring line breaks for symbols placed along lines

### DISCUSSION THREAD

**Discussion by DanielForniessoria-TomTom** [maplibre/maplibre-gl-js#3805] [Ideas]
Hi, 
Should this be fixed in MapLibre as well?
https://github.com/mapbox/mapbox-gl-js/pull/12377

**Comment by HarelM** [comment]
We can't copy code from mapbox, but it would be great if someone would open a PR to fix this without looking at the code changes in mapbox.

**Reply by HarelM** [reply]
cc: @ChrisLoer

**Comment by wipfli** [comment]
@DanielForniessoria-TomTom best to open a new issue in MapLibre GL JS stating what the problem is without a reference to a Mapbox issue. Thanks for understanding!

**Reply by DanielForniessoria-TomTom** [reply]
Hi @wipfli 

I've created this issue:
https://github.com/maplibre/maplibre-gl-js/issues/3814

I'll try to re-create screenshots with it (I lost the changes triggering the bug) but if anybody also can reproduce and share videos/screenshots, please do!
Thanks!
